### ISSUE TITLE
Material You + Material Design 3

### ISSUE THREAD

**Issue by ThatOneCalculator** [ouchadam/small-talk#111]
Because Small Talk is being developed with Kotlin and Jetpack compose, it'd be nice to have a proper native-feeling client for Matrix

**Comment by ouchadam** [comment]
Agreed! There was a fork that started the update, hopefully it wouldn't be too much work to take it across the line 🤞

**Comment by ThatOneCalculator** [comment]
I took a look at the fork... honestly I think that MD3/MY should be handled first party since the person who forked it hasn't been on github in a while
